src/__tests__/starWarsData.js

```diff
@@ -1,10 +1,11 @@
-/**
+/** @flow
  *  Copyright (c) 2015, Facebook, Inc.
  *  All rights reserved.
  *  LICENSE file in the root directory of this source tree. An additional grant
  *  of patent rights can be found in the PATENTS file in the same directory.
  */
-
+/* eslint quote-props: ["error", "as-needed",
```

Comment:
This seems unnecessary and distracts from the purpose of the PR, as does changing the quoted props below. Was this necessary?